tmk_core/avr.mk

```diff
@@ -183,6 +183,10 @@ avrdude: $(BUILD_DIR)/$(TARGET).hex check-size
 		done; \
 		echo ""; \
 		echo "Detected controller on USB port at $$USB"; \
+		if grep -q -s 'MINGW\|MSYS' /proc/version; then \
+			USB=`echo "$$USB" | perl -pne 's/ttyS(\d+)/COM.($$1+1)/e'`; \
```

Comment:
This fixes the flashing issue:

`USB=`echo "COM"$$(($$(echo $$USB | sed -e 's/\/dev\/ttyS\([0-9]*\)/\1/g') + 1))`; \`

Reply:
This would fix #2099

Reply:
Cool, I'll test on my machine and make the change if it works.

Reply:
Well, I tested if before posting it here, but please do make sure it works, as well!

Reply:
I also checked the script. It might work.
https://github.com/qmk/qmk_firmware/issues/2099#issuecomment-365719393